### PR TITLE
Add single colour version

### PR DESCRIPTION
**Pull request by stefenfriesen**
Solid black version of the logo in SVG. Bubbles have been subtracted from the liquid path for transparency. Opaque and transparent PNGs provided. I apologize for the mess of the previous PR.

### CODE REVIEW

**Comment by parkr** [comment]
*Excellent* work! I really like this. @cobyism, would you mind giving a :+1: if you like it?

**Comment by cobyism** [comment]
:+1: Yeah, this is hot. My only feedback is that I'm slightly distracted by the tail of the last `L` overlapping the test tube's outline. In the colour versions there's depth, so the overlap isn't an issue, but for this single colour one I think the test tube outline would be better unbreached. Do you both agree?

**Comment by stefenfriesen** [comment]
Definitely agree. I didn't want to change your design in any way but to make it solid so I left that alone. Another possible improvement would be to shift the test tube to the right a little. The stroke consumed what little padding was there in the original logo and now the word and tube are a bit close.

**Comment by cobyism** [comment]
Cool, yeah I wouldn't worry about not modifying the original. Those tweaks to make it work better for this treatment would definitely be worth trying. Have a play and see what looks most "right" 😊 

Also, don't be afraid to suggest changes to the colour version too, if you have suggestions!

**Comment by stefenfriesen** [comment]
Moving the test tube tackles both of those issues. Thoughts?

jekyll-logo-black

**Comment by cobyism** [comment]
:ok_hand: :sparkles: :tropical_fish:

**Comment by stefenfriesen** [comment]
Awesome. I'll update the PR. Also, I'm just going to leave this here while I'm at it...

jekyll-logo-black-red

**Comment by cobyism** [comment]
:heart_eyes_cat: I wondered if a two-colour variation was going to come up in this discussion. Fancy adding the files for that version too? :yum:

**Comment by stefenfriesen** [comment]
Sure thing. Do you want the two-colour in a separate SVG or add it as a separate layer with the black?

**Comment by cobyism** [comment]
I think just a separate layer is fine. As long as it’s easy for whoever to export the right files without having to tweak too much that’s what matters. :+1:

**Comment by stefenfriesen** [comment]
Any further changes required for this or is everyone happy with it?

**Comment by parkr** [comment]
@stefenfriesen Would you add a mention of it in the README? You can add your own section, like `Black and White Logo (With Red Test-Tube Variant)` or `Single-Color Version`.

**Comment by mattr-** [comment]
These are so awesome. :sparkles: :heart_eyes_cat: :sparkles:

**Comment by stefenfriesen** [comment]
Made the changes. Also corrected some grammar and typos. Couldn't help myself. Might want to double check it for accuracy.

File: README.md [hunk 1]
```diff
@@ -5,17 +5,26 @@ Logo files for [Jekyll](http://jekyllrb.com).
 
 ![](https://github.com/cobyism/jekyll-logo/blob/master/jekyll-logo-dark-solid.png?raw=true)
 ![](https://github.com/cobyism/jekyll-logo/blob/master/jekyll-logo-light-transparent.png?raw=true)
```
Comment (cobyism) [inline README.md]: Derp. Just realised this still points to my old version of this repo. We should probably update that. :sweat_smile:

File: README.md [hunk 2]
```diff
@@ -5,17 +5,26 @@ Logo files for [Jekyll](http://jekyllrb.com).
...
+#### Black & White
+
+A separate SVG contains a solid black version suitable for certain design or print applications. A variation with red liquid is provided as a separate layer.
 
 ## PNG Images
 
-I’ve also included images which can be used directly without having to much about
-with exporting them from the SVG every time. There’s two PNGs for each version (dark/light)
+I’ve also included images which can be used directly without having to muck about
+with exporting them from the SVG every time. There are two PNGs for each version (dark/light),
```
Comment (cobyism) [inline README.md]: Thanks for fixing those up :+1:

File: README.md (hunk 1 above)
Comment (stefenfriesen) [inline README.md]: Done! I was wondering about that :)

**Comment by parkr** [comment]
Excellent, excellent, thank you @stefenfriesen!

**Comment by parkr** [comment]
Boom, merged.

**Comment by stefenfriesen** [comment]
Thanks, @parkr! Pleasure working with you guys :)

**Pull request merged**